### PR TITLE
Docs: login-logout commands

### PR DESCRIPTION
**Pull request by bandantonio**
## Check yourself

- [ ] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

**Review by swapnilogale — CHANGES_REQUESTED** [review]
As discussed, let us split out this topic into individual `login` and `logout` pages

File: docs/commands/login.md [hunk 1]
```diff
@@ -1,13 +1,49 @@
 # `login`
 
-Use the `login` command to authenticate to the API registry. You must [generate a personal API key](../../workflows/personal-api-keys.md) first.
+## Introduction
+
+:::warning Note
+Before proceeding with the command, you must [generate a personal API key](../../workflows/personal-api-keys.md) first.
```
Comment (adamaltman) [inline docs/commands/login.md:6]:
```suggestion
Before proceeding with the command, [generate a personal API key](../../workflows/personal-api-keys.md).
```

(I'm trying to shorten/simplify)

File: docs/commands/login.md [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
 
 ```bash
+openapi login [--help] [--verbose] [--version]
+
 openapi login --verbose
 ```
+
+## Options
+
+option      | type      | required? | default | description
```
Comment (skadinna) [inline docs/commands/login.md:27]:
```suggestion
Option      |  Type      |  Required | Default | Description
```

File: docs/commands/login.md [hunk 3]
```diff
@@ -1,13 +1,49 @@
...
+------------|:---------:|:---------:|:-------:|------------
+`--help`    | `boolean` | no        | -       | Show help
+`--verbose` | `boolean` | no        | -       | Include addtional output
```
Comment (skadinna) [inline docs/commands/login.md:30]:
```suggestion
`--verbose` | `boolean` | no        | -       | Include additional output
```

File: docs/commands/logout.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+
+```bash
+openapi logout [--help] [--version]
+
+openapi logout --version
+```
+
+## Options
+
+option      | type      | required? | default | description
```
Comment (skadinna) [inline docs/commands/logout.md:17]:
```suggestion
Option      | Type      | Required | Default | Description
```

**Review by skadinna — CHANGES_REQUESTED** [review]

**Review by skadinna — APPROVED** [review]

**Review by swapnilogale — APPROVED** [review]
@bandantonio just a small style fix, but everthing else looks good

**Pull request merged**